src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -25491,32 +25636,36 @@ public class C
     public static bool TryGetValue(C? c, [NotNullWhen(true)] out string? s)
     {
         s = null;
-        return c; // 1
+        return c;
     }
 
     public static bool TryGetValue2(C c, [NotNullWhen(false)] out string? s)
     {
         s = null;
-        return c; // 2
+        return c;
     }
 
     public static implicit operator bool(C? c) => throw null!;
```

Comment:
nit: the test would be easier to understand if this operator were at the top. No requirement to move it in this PR as it wasn't introduced in this PR.